docs/reference.rst

```diff
@@ -72,16 +72,22 @@ Default: ``''`` (Empty string)
 The password to use with :setting:`AUTH_LDAP_BIND_DN`.
 
 
+.. setting:: AUTH_LDAP_CACHE_TIMEOUT
+.. setting:: AUTH_LDAP_GROUP_CACHE_TIMEOUT
 .. setting:: AUTH_LDAP_CACHE_GROUPS
 
-AUTH_LDAP_CACHE_GROUPS
-~~~~~~~~~~~~~~~~~~~~~~
+AUTH_LDAP_CACHE_TIMEOUT
+~~~~~~~~~~~~~~~~~~~~~~~
 
-Default: ``False``
+Default: ``0``
+
+If this value is nonzero, the value determines (in seconds) the amount of time
+user's group memberships and distinguished name are cached. Value ``0`` (the
+default) disables caching entirely.
 
-If ``True``, LDAP group membership will be cached using Django's cache
-framework. The cache timeout can be customized with
-:setting:`AUTH_LDAP_GROUP_CACHE_TIMEOUT`.
+If this setting is unchanged, its value will be derived from the deprecated
+settings `AUTH_LDAP_CACHE_GROUPS` (`True`/`False`) and
+`AUTH_LDAP_GROUP_CACHE_TIMEOUT` (in seconds), if those are set.
```

Comment:
This paragraph could use a `.. versionchanged` directive. For more details, see: http://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-versionchanged